### ISSUE TITLE
What max-age header should be used with batch queries?

### ISSUE THREAD

**Issue by markkelnar** [wp-graphql/wp-graphql-smart-cache#153]
When a batch of queries is being processed, and if any or more than one query has a saved 'max-age' header seconds, what should be used as the 'Cache-Control' max age header value for the [request](https://github.com/wp-graphql/wp-graphql-smart-cache/blob/main/src/Document/MaxAge.php#L170)?

Currently, a warning is encountered for batch request because the object is not an array when being processed for maxage.

```
PHP Warning:  Attempt to read property "queryId" on array in /var/www/html/wp-content/plugins/wp-graphql-smart-cache/src/Document/MaxAge.php on line 145
```

**Comment by markkelnar** [comment]
What if we use the global max-age setting for graphql requests?

**Comment by jasonbahl** [comment]
My thought is that it could use the lowest value of any query present, or the default. . .whichever value is lowest?

**Comment by jasonbahl** [comment]
closed by #156
